editor.html

```diff
@@ -186,5 +188,23 @@
         <script src="assets/js/jquery.js"></script>
         <script src="assets/js/main.js"></script>
         <script src="assets/js/editor.js"></script>
+        <script>
+          //function to upload txt file
+          var fileUploader = document.getElementById("fileUploader");
+          function uploaderFunction(){fileUploader.click();};
+          fileUploader.addEventListener("change", function readfiles(files) {
+          if (this.files && this.files[0]) {
+            var myFile = this.files[0];
+            document.querySelector("#uploadedFname").innerText = myFile.name;
+            var reader = new FileReader();
```

Comment:
Can you call this from editor.js?

Reply:
Actually there are some errors in editor.js , 
U can see in current website that word counter is not working, And so i had to make it in html

Reply:
Okay. Would you like to give a PR for fixing the script bugs?